src/main/java/madstax/controller/Controller.java

```diff
@@ -1,13 +1,21 @@
 package madstax.controller;
 
-import madstax.model.navigation.Navigator;
+import madstax.controller.navigation.NavigationListener;
+import madstax.controller.navigation.Navigator;
+import madstax.view.ApplicationWindow;
 import madstax.view.Screen;
 
-public abstract class Controller<E extends Screen> {
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+
```

Comment:
I might be misunderstanding but does importing everything from awt also import action event anyway?

Reply:
Nopes! It imports all the files under `/awt` but ignores the content inside directories such as `/awt.event`